spec/system/jobseekers_can_create_a_job_alert_from_a_search_spec.rb

```diff
@@ -68,28 +68,6 @@
         end
       end
     end
-
```

Comment:
I have removed a lot of tests that assert that we redirect to where the user previously was after they confirm their account as it is no longer the desired behaviour.

Reply:
I was puzzled by this. It makes sense. Thanks for the comment!